Google.PowerShell/Compute/GceInstanceCmdlets.cs

```diff
@@ -630,11 +631,21 @@ protected override void ProcessRecord()
 
             instance.MachineType = GetMachineTypeUrl(instance.MachineType);
             InstancesResource.InsertRequest request = Service.Instances.Insert(instance, Project, Zone);
-            Operation operation = request.Execute();
-            AddZoneOperation(Project, Zone, operation, () =>
+            try
             {
-                WriteObject(Service.Instances.Get(Project, Zone, instance.Name).Execute());
-            });
+                Operation operation = request.Execute();
+                AddZoneOperation(Project, Zone, operation, () =>
+                {
+                    WriteObject(Service.Instances.Get(Project, Zone, instance.Name).Execute());
+                });
+            }
+            catch (GoogleApiException apiEx) when (apiEx.HttpStatusCode == HttpStatusCode.Conflict)
```

Comment:
Should this also be based on 'SilentlyContinue'?

Reply:
@iantalarico What do you mean by that?

Reply:
@iantalarico This is silently continuing now. `WriteResourceExistsError(...)` writes an error record to the error stream (a suppressible non-terminating error). If we let the exception bubble up, the error would be a terminating error.

Reply:
Do we also need to check 

... when(apiEx.HttpStatusCode == HttpStatusCode.Conflict && SilentlyContinue)?

Reply:
@iantalarico We don't need to check for the `SilentlyContinue` parameter. It's a built-in PowerShell parameter that will suppress error record (which is written out by the cmdlet in the catch clause).

Reply:
Ah I see, we just write the error and let PS handle it.  SGTM